### PR TITLE
feat: add s3 bucket override to installer spec

### PR DESCRIPTION
**Pull request by DanStough**
PR 1 of 3-ish.

Added the the new field `s3Override` field to the kurl web api per design doc.

Pretty sure this will break install.sh if anyone tried to use the new field, but considering it's not really for general, use I'd rather have the review now and have someone push back on merging.

### CODE REVIEW

File: web/src/installers/index.ts [hunk 1]
```diff
@@ -10,13 +10,15 @@ import { MysqlWrapper } from "../util/services/mysql";
 import { instrumented } from "monkit";
 import { logger } from "../logger";
 import { Forbidden } from "../server/errors";
+import { consoleReporter } from "replicated-lint/dist/cmdutil/reporters";
```
Comment (areed) [inline web/src/installers/index.ts:13]: Is this needed?

File: web/src/test/installers.ts [hunk 2]
```diff
@@ -506,6 +538,14 @@ spec:
           expect(out).to.equal(undefined);
         });
       });
+
+      describe("unknown versions w/ overrides", () => {
+        it("=> void", () => {
+          const out = Installer.parse(overrideUnknownVersion).validate();
+          console.log(out)
```
Comment (areed) [inline web/src/test/installers.ts:545]: Was this for dev only?

**Review by areed — APPROVED** [review]

File: web/src/installers/index.ts (hunk 1 above)
Comment (DanStough) [reply web/src/installers/index.ts:13]: Nope, removing.

File: web/src/test/installers.ts (hunk 2 above)
Comment (DanStough) [reply web/src/test/installers.ts:545]: Yeah, also removing.

**Pull request merged**